packer/ansible/roles/foreman/files/Procfile

```diff
@@ -2,8 +2,8 @@
 
 # SUDO must be enabled in the terminal before running nf start (or run sudo nf start)
 # See .env for environment variable definitions
-graph: /bin/bash -c "echo running $TASKGRAPHDIR/index.js; cd $TASKGRAPHDIR && exec sudo node index.js"
-http: /bin/bash -c "echo running $HTTPDIR/index.js; cd $HTTPDIR && exec sudo node index.js"
-dhcp: /bin/bash -c "echo running $DHCPDIR/index.js; cd $DHCPDIR && exec sudo node index.js"
-tftp: /bin/bash -c "echo running $TFTPDIR/index.js; cd $TFTPDIR && exec sudo node index.js"
-syslog: /bin/bash -c "echo running $SYSLOGDIR/index.js; cd $SYSLOGDIR && exec sudo node index.js"
+graph: /bin/bash -c "echo running $TASKGRAPHDIR/index.js; cd $TASKGRAPHDIR && exec sudo node index.js | tee /tmp/taskgraph.log"
+http: /bin/bash -c "echo running $HTTPDIR/index.js; cd $HTTPDIR && exec sudo node index.js | tee /tmp/taskgraph.log"
```

Comment:
Are you intending to aggregate all logs to the same file? If so, perhaps rackhd.log or something is a better name?